addons/point_of_sale/static/src/js/Screens/PaymentScreen/PaymentLine.js

```diff
@@ -0,0 +1,27 @@
+odoo.define('point_of_sale.PaymentLine', function(require) {
```

Comment:
Why doing this? it create a lot of useless diff, it just renames line in _line

Reply:
One of my early changes. I'll return the original component replaced by this.